### ISSUE TITLE
Warn that the application should not be swiped away

### ISSUE THREAD

**Issue by marmistrz** [ChatSecure/ChatSecure-iOS#991]
Many users will not be aware that this will prevent ChatSecure from working correctly, as stated here: https://github.com/ChatSecure/ChatSecure-iOS/issues/902#issuecomment-349568449

Commercial IMs warn about it, e.g. WhatsApp shows a warning: 
> Swiping to close WhatsApp can prevent you from receiving notifications

It would be a good idea to show a similar notification in ChatSecure.

**Comment by Echolon** [comment]
I agree, that we may should do that tempoarily - but in the future ChatSecure should rather try to get "Voip-App status" (or what ever helps to use that API).

Because I just imagine how they'll drop it like "what a shitty app" unfortunately....

If you do so - let them choose to not show it again with a tick button

**Comment by marmistrz** [comment]
@Echolon yes, the VoIP-app status (or whatever is needed to get around Apple's silly restrictions) should be the aim. Anyway, claiming that *WhatsApp does it* should be convincing enough for most users.

Btw. most people who use ChatSecure are most probably people asked by their geek friends, so I wouldn't care about the *what a shitty app* argument. It's more the thing that they may accidentally swipe it away and miss IM messages, getting frustrated that they have not received an important one.

**Comment by chrisballinger** [comment]
When and how does WhatsApp display that notification?

**Comment by marmistrz** [comment]
@chrisballinger sorry for the delay. The friend of mine who uses an iPhone described it this way:

> after sliding the app up to close it, a notification would appear
> it would say that closing the app might result in not receiving notifications
> but apparently this notification does not appear now
